src/main/java/Duke.java

```diff
@@ -1,10 +1,280 @@
+import java.io.OutputStream;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    public static void main(String[] args){
+        // Greet
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        // Creates list
+        ArrayList<Task> list = new ArrayList<>();
+
+        //Destination
+        String dest = "data/duke.txt";
```

Comment:
Perhaps can just use destination as the variable name since it is not too long?

Reply:
I agree, destination is not long enough to need to shorten to dest.